DataFormats/FTLDigi/interface/FTLDigiCollections.h

```diff
@@ -28,7 +28,7 @@ namespace mtdhelpers {
   };
 }  // namespace mtdhelpers
 
-typedef FTLDataFrameT<FastTimeDetId, FTLSample, mtdhelpers::FTLRowColDecode> FTLDataFrame;
+typedef FTLDataFrameT<MTDDetId, FTLSample, mtdhelpers::FTLRowColDecode> FTLDataFrame;
```

Comment:
@bsunanda I am not sure whether this is needed, since the collections used are always specialized for either BTL or ETL

Reply:
May be this typedef is not strongly needed but in general the modification is OK.